### DISCUSSION TITLE
How to specify the template used on invoices

### DISCUSSION THREAD

**Discussion by hyssop** [kimai/kimai#4390] [Questions & Answers] [answered]
Once multiple invoice templates are created, how can we specify which invoice is used? I know how to change the template for individual invoices, but initially the first template in the sort order is used. This is a problem when there are two or more types of invoices that are commonly used and you are doing monthly billing (it is not fun to have to change many of them one at a time).

**Comment by kevinpapst** [comment, marked as answer]
You can attach a default template to every customer

**Reply by hyssop** [reply]
Thank you! I hadn't see that and had been search in the invoice docs.

Have you considered adding an option to select (overriding all else) the template for all invoices in the Invoice screen? Or at least a way to define the system default (instead of relying on template file name sort order)? Seems useful to me.

**Reply by kevinpapst** [reply]
That's how it was in the old version. Not useful. In some cases maybe ... but many users have templates with different languages and tax rates. And it depends on the customer which one should be used.
